telemetry/specs/INT.mdk

```diff
@@ -1462,6 +1465,7 @@ parser EgressParserImpl(packet_in packet,
     state parse_tcp {
         packet.extract(hdr.tcp);
         transition select(hdr.ipv4.dscp) {
+	    /* &&& is a concatenation operator in p4_16 */
```

Comment:
A couple of places in the P4_16 spec call `&&&` a mask operator.  Concatenation is denoted with the `++` operator in P4_16.

Reply:
Thanks. I actually used it for masking, and put a wrong comment.